File: app/java/src/main/java/com/example/production/domain/model/purchase/MiscellaneousItem.java
package com.example.production.domain.model.purchase;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class MiscellaneousItem {
    private Integer id;
    private String purchaseOrderNumber;
    private Integer lineNumber;
    private String itemCode;
    private String itemName;
    private String specification;
    private String drawingNumberMaker;
    private String version;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: app/csharp/src/ProductionManagement.Domain/Models/Purchase/MiscellaneousItem.cs
namespace ProductionManagement.Domain.Models.Purchase;

/// <summary>
/// 諸口品目情報（マスタに登録されていない臨時品目）
/// </summary>
public class MiscellaneousItem
{
    public int Id { get; set; }
    public required string PurchaseOrderNumber { get; set; }
    public required int LineNumber { get; set; }
    public required string ItemCode { get; set; }
    public required string ItemName { get; set; }
    public string? Specification { get; set; }
    public string? DrawingNumberMaker { get; set; }
    public string? Version { get; set; }
    public DateTime? CreatedAt { get; set; }
    public DateTime? UpdatedAt { get; set; }
}
